### ISSUE TITLE
Is Initiative Broken or Just Not Being Used?

### ISSUE THREAD

**Issue by discuit** [yochaigal/Cairn-FoundryVTT#16]
Hi, and thank you for providing this wonderful tool!

Currently initiative rolling doesn't seem to work in the combat tracker. It would be fantastic if this could be fixed/added.

Thank you again!

**Comment by yochaigal** [comment]
There is no initiative in Cairn.
I suppose you could track which side's turn it is, but that wouldn't be that helpful.

**Comment by discuit** [comment]
> There is no initiative in Cairn.
> I suppose you could track which side's turn it is, but that wouldn't be that helpful.

Isn't initiative based on a dex check to see if if the players act before the monsters which then becomes the cycle of combat? Much like other systems? Or did I misunderstand that?

**Comment by yochaigal** [comment]
Yes, you are correct. You however only do this once - the first round - to see _which_ PCs go before the monsters. After that it's just side vs side, the players going in any order. There is a semi-useful entry in the [FAQ](https://cairnrpg.com/resources/frequently-asked-questions/#how-is-turn-order-decided) about it. 

I've honestly never had any issue remembering which side needs to go in my FoundryVTT games, but I don't see why we shouldn't do it, certainly!

Sounds like a good feature request to me!

**Comment by discuit** [comment]
Honestly, after reading the FAQ (thank you for the link as there is a LOT of resources on that page!) you're probably right, it isn't really necessary. Would it be helpful to some? Maybe? I certainly wouldn't make it a priority though. Thanks for the prompt feedback on the matter. :)
